controllers/imageclusterinstall_controller.go

```diff
@@ -709,7 +684,7 @@ func (r *ImageClusterInstallReconciler) writeImageDigestSourceToFile(imageDigest
 
 	imageDigestMirrorSet := &apicfgv1.ImageDigestMirrorSet{
 		TypeMeta: metav1.TypeMeta{
-			APIVersion: apicfgv1.SchemeGroupVersion.String(),
+			APIVersion: apicfgv1.GroupVersion.String(),
```

Comment:
What does this change do? I imagine it's not related to the kubeadmin password, right?

Reply:
True, it's just a nit, SchemeGroupVersion was deprecated in favor of GroupVersion.
I'll move it to a separate commit